tests/testcases/mod.rs

```diff
@@ -103,6 +103,95 @@ pub async fn key_value_validation_works(controller: &dyn Controller) {
     tc.try_run(controller).await.unwrap();
 }
 
+pub async fn sqlite_works(controller: &dyn Controller) {
+    async fn checks(
+        metadata: AppMetadata,
+        test_init_key: String,
+        test_init_value: String,
+        _: Option<Pin<Box<dyn AsyncBufRead>>>,
+        _: Option<Pin<Box<dyn AsyncBufRead>>>,
+    ) -> Result<()> {
+        assert_http_response(
```

Comment:
may be use `assert_status`?

Reply:
This was copied from `key_value_works`... I will merge as stands (since you have approved and the phrasing on the comment sounds non-blocking), but am happy to update both places in a follow-up PR if you feel it's worth it.  (`assert_status` does seem terser I agree.)